.buildkite/docker-build-push.nix

```diff
@@ -42,7 +42,8 @@ with hostPkgs.lib;
 
 let
   images = map impureCreated [
-    nodePackages.dockerImage
+    nodePackages.dockerImage.testnet
```

Comment:
This should be pushing the standard image too, not just the configured public networks

Reply:
I believe that as of commit 6a5b233 the standard image is provided now via `dockerImage.custom`.